bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/templates/FormBrowser.java

```diff
@@ -64,17 +101,26 @@ public Control getControl() {
 		return container;
 	}
 
-	public void setText(String text) {
-		this.text = text;
+	public void setText(String text)
+	{
+		this.text = convertMarkdownToHtml(text);
```

Comment:
We can move this below to include within the condition if (formText != null) {...}

Reply:
@kolipakakondal 
done:
https://github.com/espressif/idf-eclipse-plugin/blob/db760d63e4174b294d5c8753cebe7ffced8ccdc1/bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/templates/FormBrowser.java#L104-L111